src/video/cameradevice.cpp

```diff
@@ -55,7 +55,10 @@ CameraDevice* CameraDevice::open(QString devName, AVDictionary** options)
     CameraDevice* dev = openDevices.value(devName);
     int aduration;
     if (dev)
-        goto out;
```

Comment:
I don't want start holy war about "goto". Many people absolutely against goto statement[1]. Why don't abstain if we can?

[1] [Edgar Dijkstra: Go To Statement Considered Harmful ](https://files.ifi.uzh.ch/rerg/arvo/courses/kvse/uebungen/Dijkstra_Goto.pdf)